src/services/Forum/FetchPreviousPosts.js

```diff
@@ -0,0 +1,55 @@
+import { firebase } from "../Firebase/Config";
+
+export async function FetchPreviousPosts(callbackSuccess) {
+  const currentUID = firebase.auth().currentUser.uid;
+  let db = firebase.firestore();
+
+  await db
```

Comment:
Should have use onSnapshot() instead of get() to listen to the firebase document.

```javascript
db.collection("users").doc(currentUID).collection("posts").onSnapshot(/* Everything else is the same */)
```